sqlmesh/core/context.py

```diff
@@ -1598,17 +1598,17 @@ def rollback(self) -> None:
 
     @python_api_analytics
     def create_external_models(self) -> None:
-        """Create a schema file with all external models.
+        """Create a file to document the schema of external models.
 
-        The schema file contains all columns and types of external models, allowing for more robust
-        lineage, validation, and optimizations.
+        The external models file contains all columns and types of external models, allowing for more
+        robust lineage, validation, and optimizations.
         """
         if not self._models:
             self.load(update_schemas=False)
 
         for path, config in self.configs.items():
-            create_schema_file(
-                path=path / c.SCHEMA_YAML,
+            create_external_models_file(
```

Comment:
To preserve compatibility with existing projects should we check whether a file with the old name already exists and use the old name if it does?